battle-engine.js

```diff
@@ -3860,8 +3860,13 @@ Battle = (function () {
 		switch (decision.choice) {
 		case 'start':
 			// I GIVE UP, WILL WRESTLE WITH EVENT SYSTEM LATER
-			var beginCallback = this.getFormat().onBegin;
-			if (beginCallback) beginCallback.call(this);
+			var format = this.getFormat();
+
+			if (format.onBegin) format.onBegin.call(this);
+			if (format.teamLength && format.teamLength.battle) {
+				this.p1.pokemonLeft = this.p1.pokemon.length = format.teamLength.battle;
```

Comment:
I am not sure this is worth 4 lines... (I'd certainly not call a function in a one-line double assignment).

I guess that will make it 5, for a summarizing comment.